src/client/camera.h

```diff
@@ -133,6 +135,22 @@ class Camera
 		return MYMAX(m_fov_x, m_fov_y);
 	}
 
+	// Returns a lambda that when called with an object's position (in BS space)
+	// returns true if, and only if the object should be frustum-culled.
+	// radius is the radius of the object.
+	auto getFrustumCuller(f32 radius) const
```

Comment:
(Might be useful in a later PR for frustum culling other objects, like CAOs, btw..)